arangod/Graph/PathManagement/PathValidator.h

```diff
@@ -112,6 +114,12 @@ class PathValidator {
       _uniqueEdges;
 
   PathValidatorOptions _options;
+  bool _isSatelliteLeader;
+
+// todo clean up all uses of USE_ENTERPRISE
+#ifdef USE_ENTERPRISE
+  std::string_view _smartValue;
+#endif
```

Comment:
```suggestion
```

It should not be a member.
Also I think it actually is never used.

Reply:
Removed.